src/status_im/contexts/keycard/migrate/fail/view.cljs

```diff
@@ -0,0 +1,36 @@
+(ns status-im.contexts.keycard.migrate.fail.view
+  (:require [quo.core :as quo]
+            [react-native.core :as rn]
+            [status-im.common.resources :as resources]
+            [utils.i18n :as i18n]
+            [utils.re-frame :as rf]))
+
+(defn view
+  []
+  (let [profile-name        (rf/sub [:profile/name])
+        profile-picture     (rf/sub [:profile/image])
+        customization-color (rf/sub [:profile/customization-color])]
+    [:<>
+     [quo/page-top
+      {:title           (i18n/label :t/failed-to-migrate-key-pair)
+       :description     :context-tag
+       :context-tag     {:full-name           profile-name
+                         :profile-picture     profile-picture
+                         :customization-color customization-color}
+       :container-style {:margin-top 56}}]
+     [rn/view {:style {:flex 1 :align-items :center :justify-content :center}}
+      [rn/image
+       {:resize-mode :contain
+        :source      (resources/get-image :keycard-migration-failed)}]]
+     [quo/divider-label (i18n/label :t/what-you-can-do)]
+     [rn/view {:style {:padding-horizontal 10}}
+      [quo/markdown-list
+       {:container-style {:padding-vertical 10}
+        :description     (i18n/label :t/log-out-remove-profile)}]
+      [quo/markdown-list
+       {:container-style {:padding-bottom 25}
```

Comment:
These manual paddings in list items are repeated here and in other namespaces. Are we correctly following Figma and the design system? I made a separate comment down below in more detail.